setup.cfg

```diff
@@ -6,5 +6,8 @@ license_file = LICENSE.txt
 # This flag says to generate wheels that support both Python 2 and Python
 # 3. If your code will not run unchanged on both Python 2 and 3, you will
 # need to generate separate wheels for each Python version that you
-# support.
+# support. Setting the value of 'universal' to anything other than '1',
+# 'true' or 'yes' (case insensitive) or removing this line will prevent
+# bdist_wheel from trying to make a universal wheel. For more see:
+# https://packaging.python.org/tutorials/distributing-packages/#wheels
```

Comment:
Trying to give the whole story here seems likely to confuse people. I'd prefer the addition to just say "Removing this line (or setting universal to 0) will prevent..."

Reply:
Thanks! I agree, your version is better.